src/frontend/components/SubscribeScheduledVideoEmailForm/index.spec.tsx

```diff
@@ -0,0 +1,291 @@
+import {
+  act,
+  fireEvent,
+  render,
+  screen,
+  waitFor,
+} from '@testing-library/react';
+import { Deferred } from '../../utils/tests/Deferred';
```

Comment:
Can you use absolute imports in place of ascending relative ones in this file please ?